internal/pkg/gzip/gunzip.go

```diff
@@ -0,0 +1,32 @@
+package gzip
+
+import (
+	"compress/gzip"
+	"os"
+	"path/filepath"
+
+	"github.com/puppetlabs/pdkgo/internal/pkg/utils"
+)
+
+func Gunzip(source, target string) error {
```

Comment:
Do we need gunzip and gzip?

Reply:
Just for the benefit of those viewing, the `Gunzip` is the method that'll extract a `.tar.gz` packaged template